1.8/administration/virtual-networks/index.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: page
+post_title: Virtual Networks
+menu_order: 11.5 
+---
+
+DC/OS allows you to create, manage, and use virtual networks, supporting IP-per-container with isolation guarantees amongst subnets. Virtual networks allow both Mesos and Docker containers to communicate from within a single node and between nodes on a cluster, and for services to be created so that their traffic is isolated from other traffic coming from any other virtual networks or hosts in the cluster.
```

Comment:
isolation guarantees among subnets depends on the CNI implementation you're using and/or firewall policies that Dan implements...